src/main/java/pl/damianszczepanik/jenkins/buildhistorymanager/descriptors/actions/DeleteArtifactsWithPatternActionDescriptor.java

```diff
@@ -0,0 +1,47 @@
+package pl.damianszczepanik.jenkins.buildhistorymanager.descriptors.actions;
+
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import hudson.Extension;
+import hudson.model.Descriptor;
+import pl.damianszczepanik.jenkins.buildhistorymanager.model.actions.Action;
+import pl.damianszczepanik.jenkins.buildhistorymanager.model.actions.DeleteArtifactsWithPatternAction;
+
+/**
+ * Descriptor implementation needed to render UI for {@link DeleteArtifactsWithPatternAction}.
+ */
+@Extension
+@Symbol("DeleteArtifactsWithPattern")
+public class DeleteArtifactsWithPatternActionDescriptor extends Descriptor<Action>{
+
+    private String include;
+    private String exclude;
+
+    public DeleteArtifactsWithPatternActionDescriptor() {
+        super(DeleteArtifactsWithPatternAction.class);
+    }
+
+    public String getInclude() {
+        return include;
+    }
+
+    @DataBoundSetter
+    public void setInclude(String include) {
+        this.include = include;
+    }
+
+    public String getExclude() {
+        return exclude;
+    }
+
+    @DataBoundSetter
+    public void setExclude(String exclude) {
+        this.exclude = exclude;
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Delete artifacts with pattern";
```

Comment:
not sure if `with` is a right word here because I think this does not describe how this action work

Reply:
`Delete artifacts matching a pattern` or `Delete artifacts matching patterns`?

Reply:
@damianszczepanik @martinda @olivierdagenais 
We are considering renaming `"Delete artifacts with pattern"`. Please cast your vote for the renaming option:

A) Delete artifacts matching pattern
B) Delete artifacts matching patterns
C) Delete artifacts by matching patterns

Please reply with your choice. Thanks!

Reply:
I vote for `Delete artifacts matching patterns`.

Reply:
Suggestion was applied. Thanks for voting.

Reply:
Voting for: `Delete artifacts matching patterns`